assets/common/common.go

```diff
@@ -80,6 +80,36 @@ func RunGitCommand(command string, formating ...interface{}) error {
 	return cmd.Run()
 }
 
+// RunGitCommand generically runs a Git command and saves output to a file in .git directory
+func RunGitCommandSaveOutputToFile(command string, outputFilename string) error {
+	args := strings.Split(command, " ")
```

Comment:
Only thing I would suggest is to create another method to avoid duplicate code with lines 77-79 and invoke this in both places.

Reply:
That crossed my mind. It's actually only lines 78 and 79 (line 77 is slightly different), which was why I was hesitant to create a new method. But let me know if you still think it's worth a new method.

Reply:
I think that formatting on line 77 is an optional parameter that can be incorporated with the new method. Not a huge concern just thought I'd point it out. Up to you whether or not to change it.

Reply:
Ah right. I'll update.